src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/CastExpressionSyntaxExtensions.vb

```diff
@@ -10,21 +10,127 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
     Friend Module CastExpressionSyntaxExtensions
         <Extension>
         Public Function Uncast(cast As CastExpressionSyntax) As ExpressionSyntax
-            Return Uncast(cast, cast.Expression)
+            Return Uncast(cast, cast.OpenParenToken, cast.Expression, cast.CommaToken, cast.Type, cast.CloseParenToken)
         End Function
 
         <Extension>
         Public Function Uncast(cast As PredefinedCastExpressionSyntax) As ExpressionSyntax
-            Return Uncast(cast, cast.Expression)
+            Return Uncast(cast, cast.OpenParenToken, cast.Expression, Nothing, Nothing, cast.CloseParenToken)
         End Function
 
-        Private Function Uncast(castNode As ExpressionSyntax, innerNode As ExpressionSyntax) As ExpressionSyntax
-            Dim resultNode = innerNode.WithTriviaFrom(castNode)
+        Private Function Uncast(castNode As ExpressionSyntax, openParen As SyntaxToken, innerNode As ExpressionSyntax, commaToken As SyntaxToken, typeNode As TypeSyntax, closeParen As SyntaxToken) As ExpressionSyntax
+
+            Dim leadingTrivia As List(Of SyntaxTrivia) = castNode.GetLeadingTrivia.ToList
+            If openParen.LeadingTrivia.ContainsCommentOrLineContinue Then
+                leadingTrivia.AddRange(openParen.LeadingTrivia)
+            End If
+
+            If openParen.TrailingTrivia.ContainsCommentOrLineContinue Then
+                leadingTrivia.AddRange(openParen.TrailingTrivia)
+            End If
+            If (Not castNode.GetLeadingTrivia.Any) AndAlso leadingTrivia.FirstOrDefault.IsKind(SyntaxKind.WhitespaceTrivia) Then
```

Comment:
Please include parens on method calls. Please to not include parens for 'if'

Reply:
Is there a editorConfig setting for VB for () after methods that is missing in Roslyn? C# requires the () but VB doesn't and you would have to catch them all manually. If not I will open an issue. I fixed the () around If it was their for clarity.